src/componets/Weather/Box/Box.module.scss

```diff
@@ -1,9 +1,9 @@
 .Box {
   padding: 16px;
   border-radius: 4px;
-  text-align: center;
-  color: #ede9f9;
-  font-weight: 700;
-  background-color: #47b0b0;
+  
+  color: #000000;
+  
+  background-color: #98FB98;
   box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
```

Comment:
use em/rem

Reply:
penge po sample 

from
$default-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
to
$default-shadow: 2rem (32px);

tama po ba?